Lecture3/Task1/utils/IOFileWriter.java

```diff
@@ -0,0 +1,16 @@
+package utils;
+
+import java.io.*;
+
+public class IOFileWriter {
+	public static PrintWriter createFile(String fileName) {
```

Comment:
let's encapsulate logic with `PrintWriter `  - no need to return this technical object

let's create method `storeProductsInFile(List<Product> products, file) ``
responsibility - create  `PrintWriter ` and  print all products and close writer in finally block

Reply:
done